pysmiles/smiles_helper.py

```diff
@@ -876,6 +877,42 @@ def _mark_chiral_atoms(molecule):
             neighbours = [neighbours[0],  neighbours[1], neighbours[3], neighbours[2]]
         molecule.nodes[node]['rs_isomer'] = tuple(neighbours)
 
+def check_for_ez_conflicts(anchor, tagged_nodes, ez_isomer_class):
+    n1, n2 = tagged_nodes
+    if (n1 < anchor and n2 < anchor) or (n1 > anchor and n2 > anchor):
+        assert ez_isomer_class[n1] != ez_isomer_class[n2]
+    else:
+        assert ez_isomer_class[n1] == ez_isomer_class[n2]
```

Comment:
I think it would be better to raise something (`ValueError`?) with a sane message.
Also, please make this function private, and maybe add a short docstring